docker-compose.yml

```diff
@@ -77,16 +67,34 @@ services:
       URL_WS_SERVER_SCHEME: ${URL_WS_SERVER_SCHEME:-ws://}
       URL_WS_SERVER_HOST: ${URL_WS_SERVER_HOST:-}
       URL_WS_SERVER_PORT: ${URL_WS_SERVER_PORT:-${WS_SERVER_PORT:-45001}}
-      URL_WS_SERVER_URI: ${URL_WS_SERVER_URI:-}
+      URL_WS_SERVER_URI: ${URL_WS_SERVER_URI:-/ws/}
```

Comment:
same fix as in 
https://github.com/BredaUniversityResearch/MSPChallenge-Server/pull/193/files